src/masa_internal.h

```diff
@@ -2070,6 +2070,79 @@ class ad_cns_3d_crossterms : public manufactured_solution<Scalar>
 
 
 // ------------------------------------------------------
+// --------------- ad_cns_3d_les
+// ------------------------------------------------------
```

Comment:
Would you add a reference to the location of the source documentation here? Might be as simple as,

"// Source Documentation available at: analytical/navier_stokes/navier_stokes_comp_steady_les/latex"

After that I'll merge.